Added support for Romanian translation

Achime11 opened the pull request.

Coding-Hen commented on Nitrox.Subnautica.Assets/LanguageFiles/Romanian.json:
```diff
@@ -0,0 +1,37 @@
+	"Nitrox_EnterName": "Introdu numele de jucător",
+	"Nitrox_SubmitPassword": "Trimite parola",
+	"Nitrox_MultiplayerLoaded": "Clientul Multiplayer încărcat ...",
+	
```
Minor: remove spaces between as to keep consistency with other language files

Achime11 replied:
Fixed

Coding-Hen commented on Nitrox.Subnautica.Assets/LanguageFiles/Romanian.json:
```diff
@@ -0,0 +1,37 @@
+	"Nitrox_JoiningSession": "Alăturarea la sesiune Multiplayer",
+	"Nitrox_SyncingWorld": "Sincronizarea Lumii Multiplayer",
+	"Nitrox_LostConnection": "Conexiune pierdută la serverul jocului",
+	"Nitrox_PlayerKicked": "Ai fost dat afară de pe server",
+		
+	// In game logging
```
```suggestion
	"Nitrox_PlayerKicked": "Ai fost dat afară de pe server",
	// In game logging
```

Coding-Hen commented on Nitrox.Subnautica.Assets/LanguageFiles/Romanian.json:
```diff
@@ -0,0 +1,37 @@
+	"Nitrox_PlayerColor": "Culoarea jucătorului",
+	"Nitrox_EnterName": "Introdu numele de jucător",
+	"Nitrox_SubmitPassword": "Trimite parola",
+	"Nitrox_MultiplayerLoaded": "Clientul Multiplayer încărcat...",
+	
+	// In game menu
```
```suggestion
	"Nitrox_MultiplayerLoaded": "Clientul Multiplayer încărcat...",
	// In game menu
```

dartasen commented:
Hi, thanks for the translation.
We updated the translation system in #1364 and we added new translations, is it possible to update this translation with the new entries ?

See [New english file](https://github.com/dartasen/Nitrox/blob/translations/Nitrox.Subnautica.Assets/LanguageFiles/English.json)
Those were added : 
```json
"Nitrox_JoinServerPasswordHeader": "Server password required",
"Nitrox_DisconnectedSession": "Disconnected from server",
"Nitrox_FirewallInterfering": "Seems like your firewall settings are interfering",
```

Coding-Hen reviewed (changes requested):
Sorry I didnt completly explain. Lets remove these spaces to be consistent with #1364

Achime11 commented:
> Hi, thanks for the translation.
> We updated the translation system in #1364 and we added new translations, is it possible to update this translation with the new entries ?
> 
> See [New english file](https://github.com/dartasen/Nitrox/blob/translations/Nitrox.Subnautica.Assets/LanguageFiles/English.json)
> Those were added :
> 
> ```json
> "Nitrox_JoinServerPasswordHeader": "Server password required",
> "Nitrox_DisconnectedSession": "Disconnected from server",
> "Nitrox_FirewallInterfering": "Seems like your firewall settings are interfering",
> ```

Coding-Hen reviewed (changes requested):
Lets unindent this to keep consistant formatting with #1364
